pkgs/development/tools/wp4nix/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitLab, nix, subversion }:
+
+buildGoModule rec {
+  pname = "wp4nix";
+  version = "1.0.0";
```

Comment:
TIL we tagged a release

Reply:
same, tbh

Reply:
I guess we were referencing this version tag https://git.helsinki.tools/helsinki-systems/wp4nix/-/tags
Maybe it would be better to use an unstable version tag?